cherry_picker/readme.rst

```diff
@@ -53,6 +57,19 @@ repository are pushed to ``origin``. If this is incorrect, then the correct
 remote will need be specified using the ``--pr-remote`` option (e.g.
 ``--pr-remote pr`` to use a remote named ``pr``).
 
+Personal Access Token (optional)
+--------------------------------
+
+``cherry_picker`` can automatically create the PR, if the ``GH_AUTH`` token is set
```

Comment:
Is this considered safe and best practice for storing an auth token like this? I honestly don't know, which is why I'm asking. 😄

Reply:
I don't really know 😅 
This is really so I can use it for the cherry picker bot later.
Maybe I should keep this as hidden and secret functionality, and not mention it at all in the readme?

Reply:
Yeah, either don't document it or be clear that one should be careful about how that environment variable is stored.

Reply:
Ok I'm going with "not documenting it" since it's easier to do than coming up with warning messages 😅